cypress/page/login.page.ts

```diff
@@ -18,7 +18,8 @@ class LoginPage {
         cy.get(this.passInput).type(pass)
     }
 
-    public login(): void {
+    public login
```

Comment:
leave the `public login(): void {` on the same line

Reply:
Done